openapi/workflow_execution_service.openapi.yaml

```diff
@@ -788,14 +867,67 @@ components:
         task_logs:
           type: array
           items:
-            $ref: '#/components/schemas/Log'
+            $ref: '#/components/schemas/TaskLog'
           description: The logs, and other key info like timing and exit code, for each step in the workflow run.
         next_page_token:
           type: string
           description: >-
             A token which may be supplied as `page_token` in workflow run task list request to get the next page
             of results.  An empty string indicates there are no more items to return.
       description: The service will return a TaskListResponse when receiving a successful TaskListRequest.
+    TaskLog:
```

Comment:
Since this seems to be strictly a superset of `Log` and is also _intended_ to be a superset of `Log` (at least until a hypothetical future in which this functionality would be provided by TES or externalized, generic WES/TES schemas; which would require breaking changes anyway), shouldn't we then rather make use of `allOf` to _extend_ the existing `Log` schema with the new properties and thus avoid repetition?